chapters/ch33.md

```diff
@@ -1,52 +1,50 @@
 
 
 > There are some tasks that even humans aren’t good at. For example, picking a book to recommend to you; or picking an ad to show a user on a website; or predicting the stock market. Computers already surpass the performance of most people on these tasks. With these applications, we run into the following problems:
 
---> _replace THIS LINE by your translation for the above line_
+ Có những tác vụ mà ngay cả con người cũng không giỏi. Ví dụ, chọn một cuốn sách để giới thiệu cho bạn; hoặc chọn một quảng cáo để hiển thị cho người dùng trên một trang web; hoặc dự đoán thị trường chứng khoán. Máy tính đã vượt qua hiệu suất của hầu hết mọi người trong các tác vụ này. Với các ứng dụng này, chúng ta gặp phải các vấn đề sau:
 
 > * It is harder to obtain labels.​ For example, it’s hard for human labelers to annotate a database of users with the “optimal” book recommendation. If you operate a website or app that sells books, you can obtain data by showing books to users and seeing what they buy. If you do not operate such a site, you need to find more creative ways to get data.
 
---> _replace THIS LINE by your translation for the above line_
+* Việc lấy nhãn khó hơn.​ Ví dụ, người ghi nhãn khó có thể dán nhãn một cơ sở dữ liệu người dùng với danh sách gợi ý sách tối ưu. Nếu bạn vận hành một trang web hoặc ứng dụng bán sách, bạn có thể lấy dữ liệu bằng cách hiển thị sách cho người dùng và xem những gì họ mua. Nếu bạn không vận hành một trang web như vậy, bạn cần tìm nhiều cách sáng tạo hơn để lấy dữ liệu.
 
 
 > * Human intuition is harder to count on.​ For example, pretty much no one can predict the stock market. So if our stock prediction algorithm does no better than random guessing, it is hard to figure out how to improve it.
 
---> _replace THIS LINE by your translation for the above line_
+ * Trực giác của con người khó dựa vào hơn. Ví dụ, khá nhiều người không thể dự đoán được thị trường chứng khoán. Vì vậy, nếu thuật toán dự đoán cổ phiếu của chúng ta không tốt hơn đoán ngẫu nhiên, thật khó để tìm ra cách cải thiện nó.
 
 
-> * It is hard to know what the optimal error rate and reasonable desired error rate is. ​Suppose you already have a book recommendation system that is doing quite well. How do you know how much more it can improve without a human baseline?
 
---> _replace THIS LINE by your translation for the above line_
+> * It is hard to know what the optimal error rate and reasonable desired error rate is. ​Suppose you already have a book recommendation system that is doing quite well. How do you know how much more it can improve without a human baseline?
 
+ * Thật khó để biết tỷ lệ lỗi tối ưu và tỷ lệ lỗi mong muốn hợp lý là gì. ​ Giả sử bạn đã có một hệ thống giới thiệu sách đang hoạt động khá tốt. Làm thế nào để bạn biết nó có thể cải thiện bao nhiêu nếu không có nền tảng của con người?
```

Comment:
* Hai dấu cách sau dấu chấm câu thứ nhất.
* Từ "baseline" trong machine learning không hẳn có nghĩa là "nền tảng". Mình chưa biết nên dịch từ này như thế nào, nhưng nghĩa của nó là một giải pháp đơn giản của một bài toán để biết được giải pháp của bạn tốt thế nào. Trong đoạn này, ý tác giả là nếu không có một cái mốc cụ thể (ở đây là kết quả dựa vào con người) thì làm sao đoán được hệ thống có thể cải thiện được bao nhiêu.

Reply:
Em hiểu ý của anh ạ, em cũng chưa tìm được từ nào ngắn gọn xúc tích, thích hợp để diễn tả từ "baseline" này. Cụm từ này có thể không dịch mà để nguyên tiếng anh không a?

Reply:
Hoặc từ "Mốc" ạ?

Reply:
Anh cũng đang hướng về việc giữ nguyên gốc tiếng Anh cho tới khi tìm được từ phù hợp. Tuy nhiên cụm "human baseline" sẽ hơi kỳ quặc nếu không dịch hết.

Reply:
Riêng từ "baseline" có thể dịch là "giải pháp cơ sở", nhưng khi đó "human baseline" vẫn khó dịch.

@sonvx @ngcthuong @william-vu có ý tưởng gì không?

Reply:
e cũng không biết dịch là gì. :)

Reply:
Mình đề thì "baseline" dịch là "giải pháp cơ sở", còn "human basline" có thể dịch gọn là "giải pháp (từ) con người".

Reply:
Tạm thời trong chương này em sẽ để là "baseline" trước khi thống nhất được cho cụm từ human baseline hoặc baseline ạ.

Reply:
" Làm thế nào để bạn biết nó có thể cải thiện bao nhiêu nếu không có giải pháp từ con người?"
Em cảm thấy từ giải pháp của a rất hay nhưng e nghĩ nếu dùng cụm từ ý thì cả câu này chưa diễn tả được hết ý nghĩa câu này muốn thế hiện :<

Reply:
Từ "baseline" em có thể giữ nguyên được, nhưng từ "human" anh thấy mình có trách nhiệm phải dịch ra :D.

Giả sử mình giữ từ baseline, vậy thì có một vài lựa chọn:
* baseline từ con người
* baseline bởi con người
* baseline con người
* baseline là con người
* ...
Em thấy cụm nào hợp lý không?

Reply:
"human baseline" a thấy có thể là "giải pháp mức con người". Từ baseline trong cụm từ "human baseline" a thấy không cần đưa từ "cơ sở" vào. Vì khác với các giải pháp từ AI, giải pháp của "human baseline" thường chỉ có 1 và được coi là groundtruth để mô hình AI hướng tới. 

Một số alternativies mọi người vote coi:
- "giải pháp tầm con người"
- "giải pháp cấp con người" (bạn @honghanhh cũng có dùng từ cấp con người trong phần dịch)
- "giải pháp như con người" -> cái này cũng khá hay.

Reply:
E đang thiên về "baseline từ con người" và "giải pháp cấp con người" ạ, không biết mn thấy sao ạ?

Reply:
@honghanhh anh vote "giải pháp cấp con người". Em thay đi rồi anh approve PR này. 

Em làm chương này rất tốt. Cảm ơn em.

Reply:
Vâng e vừa thay xong rồi ạ, Em cảm ơn ạ